includes/RequestWiki/SpecialRequestWikiQueue.php

```diff
@@ -24,6 +24,16 @@ public function execute( $par ) {
 			$this->doPagerStuff();
 		} else {
 			$this->lookupRequest( $par );
+
+			$this->getOutput()->setPageTitle( $this->msg( 'requestwikiqueue-request-view' )->text() );
```

Comment:
I'm confused what the purpose of this is?

Reply:
See comment for commit. If this is merged, both the title and subtitle will read "Wiki request queue" which is a lot of repetition.

Reply:
> See comment for commit. If this is merged, both the title and subtitle will read "Wiki request queue" which is a lot of repetition.

No, subtitle will read return to wiki request queue with my suggested changes, which is how everything in core and other extensions do it also. I don't see a reason to make that change here.

Reply:
The proposed code adds a "Return to" link at the bottom of the form.

Reply:
> The proposed code adds a "Return to" link at the bottom of the form.

Yes, if you use same placement.

Reply:
Try placing above lookupRequest

Reply:
But why not use a subtitle link as Special:ChangeCredentials, Special:GlobalPreferences, and Special:Interwiki/add do? The placement of the return to text is awkward at the bottom.

Reply:
Because I dislike how that is done. Its a mess and goes against my plans to clean it up to use core functions. I am highly opposed to doing it the current way. addReturnTo() is one method designed for this, so thats why I want to. But if you want to use a subtitle. Do it simply with $this->getOutput()->addBacklinkSubtitle( $this->getPageTitle() );

Reply:
Understandable, noted for next time.